.github/workflows/ci.yml

```diff
@@ -41,3 +41,13 @@ jobs:
       run: python3 driver.py --lint
     - name: Build, then Test
       run: python3 driver.py --test
+    - name: Set spartan field feature on
+      run: python3 driver.py --features zok smt r1cs ristretto255
+    - name: Check (spartan custom build)
+      run: python3 driver.py --check
+    - name: Format (spartan custom build)
+      run: cargo fmt -- --check
+    - name: Lint (spartan custom build)
+      run: python3 driver.py --lint
+    - name: Build, then Test (spartan custom build)
+      run: python3 driver.py --test
```

Comment:
Possibly naive question: is it possible for us to run both the bellman and spartan tests from a single build (with a single set of enabled features)?

Reply:
Not with the way the fields are currently set up. The choice of field is made at build time (selected in `src/util/field.rs` by chosen features). This field can technically be disregarded/changed after building, right before conversion of r1cs to spartan, if you are compiling a front end like C. But for a front end like Zokrates, that encodes literal field elements, it is important the field is correct from the get go.

So you can either select the bellman field OR the spartan field during the build.

Reply:
Okay, that makes sense. I think we could get around this restriction, but it'll take a bit of work.

I worry about the latency & potential cache-thrash induced by doing a double-build. I propose that this PR should merge the test script and driver modifications, but not modify the CI script. When we fix the field setup, we can broaden CI.

Does sound good to you?

Reply:
Sure!